utils/Dockerfile.llvm

```diff
@@ -0,0 +1,25 @@
+FROM ubuntu:12.04
+
+# Used to build llvm and sub-packages
+# See https://github.com/mapbox/mason/blob/master/scripts/llvm/base/README.md for details
+# docker build -t mason-llvm -f Dockerfile .
+# docker run -it mason-llvm bash
+
+# we mimic the travis path to ensure any hardcoded paths in deps
+# do not cause problem's when building (an example is freetype-config)
+ENV WORKINGDIR /home/travis/build/mapbox/mason/
+WORKDIR ${WORKINGDIR}
+
+COPY mason mason
+COPY mason.sh mason.sh
+COPY scripts scripts
+COPY utils utils
+
+RUN apt-get update -y && \
+ apt-get install -y vim python build-essential bash curl git-core ca-certificates software-properties-common python-software-properties --no-install-recommends
+
+RUN add-apt-repository -y ppa:ubuntu-toolchain-r/test && \
```

Comment:
Hm I guess pull this ppa in for a recent C++ stdlib version. But why exactly? Does building llvm or any of the binaries need a recent stdlib? We do not link to the stdlib in the binaries anyway, no? I don't quite follow here.